ocs_ci/ocs/cluster.py

```diff
@@ -394,6 +400,23 @@ def mon_change_count(self, new_count):
         logger.info(f"Mon count changed to {new_count}")
         self.cluster.reload()
 
+    def update_mon_count_to_five(self):
```

Comment:
I suggest adding the param `new_count=5` to the function signature. That way, the default value will still be 5, but we will have the option to test with another mon count value. The function signature should change to `update_mon_count(new_count=5)` in this case.